src/templates/angular/src/app/pages/users/users.component.ts

```diff
@@ -31,19 +37,34 @@ export class UsersComponent implements OnInit {
   <% if(isCrudWithNode){%>
   getUsers() {
     this.apiService.getEmployees().subscribe((res: any) => 
-      this.users = res
+      this.users = res.map((each: any)=> {
+        const refactoredUserData: object = {
+          id: each.id,
+          name: each.name,
+          username: each.username,
+          email: each.email,
+        }
+        return refactoredUserData
+        }
```

Comment:
this.users = res.map((each: any)=> ({
          id: each.id,
          name: each.name,
          username: each.username,
          email: each.email,
        })